### PR TITLE
Installed packages

### PR DESCRIPTION
**Pull request by imMichaelHarris**

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -1,8 +1,26 @@
 // we'll need axios
+import axios from "axios";
 
 // we'll need to create 3 different action types here.
 // one for fetching, one for success and one for failure
+export const FETCH_START = "FETCH_START";
+export const FETCH_SUCCESS = "FETCH_SUCCESS";
+export const FETCH_FAILED = "FETCH_FAILED";
```
Comment (Fractured2K) [inline src/actions/index.js:8]: Finite state pattern 🔥

File: src/actions/index.js [hunk 2]
```diff
@@ -1,8 +1,26 @@
...
+  setTimeout(() => {
+    axios
+    .get("https://swapi.co/api/people")
+    .then(res => {
+      dispatch({ type: FETCH_SUCCESS, payload: res.data.results });
+    })
+    .catch(err => {
+      dispatch({ type: FETCH_FAILED, payload: err });
+    });
+  }, 3000)
```
Comment (Fractured2K) [inline src/actions/index.js:24]: Curious how come there is a timeout here? I'm assuming it's to the loader?

File: src/reducers/starWarsReducer.js [hunk 3]
```diff
@@ -1,13 +1,36 @@
...
+          ...state,
+          fetching: false,
+          characters: action.payload
+        }
+
+      case FETCH_FAILED:
+        return {
+          ...state,
+          fetching: false,
+          error: `UH OH... REQUEST FAILED. - ${action.payload}`
```
Comment (Fractured2K) [inline src/reducers/starWarsReducer.js:30]: Love the way you handled your error here!

Also make sure to reset your fetching and error state on each case.

**Review by Fractured2K — APPROVED** [review]
Great job on this project Michael! Loving all the commits you're making! One thing, when committing don't make it your commits past tense, make them present tense.

_e.g_
Instead of `Passed in logger to applymiddleware`
say `Pass logger to applymiddleware`

File: src/actions/index.js (hunk 2 above)
Comment (imMichaelHarris) [reply src/actions/index.js:24]: Yeah I just put that there to see the loader it isn't necessary

**Pull request merged**